src/main/java/com/yelp/nrtsearch/server/luceneserver/search/SearchContext.java

```diff
@@ -144,6 +147,11 @@ public SharedDocContext getSharedDocContext() {
     return sharedDocContext;
   }
 
+  /** Get highlighting requirements for the request */
+  public HighlightFetchTask getHighlightFetchTask() {
+    return highlightFetchTask;
+  }
```

Comment:
nit: Would be good for the doc string to note that this will be null if no fields are highlighted